src/momento/simple_cache_client_async.py

```diff
@@ -135,6 +135,12 @@ def __init__(self, configuration: Configuration, credential_provider: Credential
                 It is possible to override this setting when calling the set method.
         Raises:
             IllegalArgumentException: If method arguments fail validations.
+        Example:
+        ::
```

Comment:
Is the double colon a formatting thing?

Reply:
The double colon is the way I've seen recommended to display code examples in docstrings. I'm using it wrong here, however. It should be Example:: followed by a newline followed by the code. I wasn't using the newline and was getting weird formatting so I tried putting it on its own line and it worked. I'll alter it to be more correct.

Reply:
Thank you! I was looking for this feature. Will update the rest of the docs accordingly.